snorkel/contrib/learning/relstm.py

```diff
@@ -147,7 +147,7 @@ def train(self, candidates, training_marginals, n_epochs=25, lr=0.01,
             @lr: learning rate
             @dim: embedding dimension
             @batch_size: batch size for mini-batch SGD
-            @rebalance: rebalance training examples?
+            @rebalance: bool or fraction of positive examples desired
```

Comment:
Just make this a tiny bit more clear- e.g. state that 0.5 would represent standard class balancing, `False` means no class balancing, what is `True`?